build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+buildscript {
+    val kotlin_version by extra("1.3.71")
+    repositories {
+        jcenter()
+        google()
+        maven { url = uri("http://oss.sonatype.org/content/repositories/snapshots/") }
+    }
+    dependencies {
+        classpath("com.vanniktech:gradle-maven-publish-plugin:0.9.0")
+        classpath("com.android.tools.build:gradle:3.6.1")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
+        classpath("org.jetbrains.dokka:dokka-gradle-plugin:0.10.0")
+        classpath("org.jacoco:org.jacoco.core:0.8.5")
+    }
+}
+
+plugins {
+    id("io.gitlab.arturbosch.detekt").version("1.7.2")
+    id("com.github.kt3k.coveralls").version("2.10.0")
+}
+
+dependencies {
+    detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.8.0")
+}
+
+allprojects {
+    repositories {
+        jcenter()
+        google()
+    }
+}
+
+tasks.register("clean", Delete::class) {
+    delete(rootProject.buildDir)
+}
+
+val detektAll by tasks.registering(io.gitlab.arturbosch.detekt.Detekt::class) {
+    description = "Runs over whole code base without the starting overhead for each module."
+    buildUponDefaultConfig = true
```

Comment:
Switched to default config flag. This is useful if you just want the default rules out of the box (new ones when updating detekt will automatically get included).

If you want to customize it copy over the latest default config from [here](https://github.com/detekt/detekt/blob/master/detekt-cli/src/main/resources/default-detekt-config.yml)

Remove this flag & add the following to both detektAll and detektAllBaseline
`config.setFrom(files("$rootDir/config/detekt/detekt.yml"))`